bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyBaseHandler.java

```diff
@@ -1006,6 +1008,12 @@ protected void initializeThingConfig() {
         config.serviceName = getString(properties.get(PROPERTY_SERVICE_NAME));
         config.localIp = bindingConfig.localIP;
         config.localPort = String.valueOf(bindingConfig.httpPort);
+        if (config.localIp.startsWith("169.254")) {
```

Comment:
@lolodomo Please advise here

Reply:
Ok, I understand. Is the address 169.254.x.x the IP that all users will get in such a case or is it something specific to your network settings ?

Reply:
169.254.x.x are self-assigned IPs when DCHP fails, it the binding uses this to configure the device's call back (websocket url) this will lead to nowehre = no incoming events

Reply:
Preferably solve (check log/detail messages) it the same as here: https://github.com/openhab/openhab-addons/pull/16306